st2api/st2api/controllers/v1/webhooks.py

```diff
@@ -218,6 +220,10 @@ def _get_headers_as_dict(self, headers):
             headers_dict[key] = value
         return headers_dict
 
+    def _filter_authentication_headers(self, headers):
+        auth_headers = [HEADER_API_KEY_ATTRIBUTE_NAME, HEADER_ATTRIBUTE_NAME, 'Cookie']
```

Comment:
Filtering out `St2-Api-Key` and `X-Auth-Token` is great.
However `Cookie` may not necessarily contain auth data, but some useful params to use and filter as payload.

@StackStorm/maintainers any opinions?

Reply:
May not but could if I understand `st2common/st2common/router.py#L278` right. Another option is to parse cookies here or pass the `req.cookies` from the router to the controller.

Reply:
This fix is marked for the upcoming `v3.3.0` release.

@StackStorm/maintainers @StackStorm/contributors any opinions about this special case? WDYT?